package-lock.json

```diff
@@ -44,6 +44,22 @@
         "node": ">=18.0.0"
       }
     },
+    "../../../algokit/algokit-utils/algokit-utils-ts-debug/dist": {
+      "name": "@algorandfoundation/algokit-utils-debug",
```

Comment:
Can we do this without adding algokit as a dependency? Strong preference to have the dependency going in solely one direction.